x-pack/solutions/security/test/security_solution_api_integration/config/services/detections_response/kibana.jsonc

```
@@ -0,0 +1,8 @@
+{
+  "type": "test-helper",
+  "id": "@kbn/security-solution-detections-response-service",
```

Nit: as this is a subpackage of the below, I think this could be named more accurately:

```suggestion
  "type": "test-helper",
  "id": "@kbn/detections-response-ftr-services",
```